PublicationsRouter/v2/sword-out/XWALK-DSpace.md

```diff
@@ -0,0 +1,115 @@
+ XML format supplied to repositories via SWORDv2.
+ 
+For information about the schema see:
+
+* [Dubin Core](http://dublincore.org/documents/dcmi-terms/)
+
+The following table lists the notification fields in the JSON, then the DC terms fields it will be transformed to, and
+any notes regarding the transformation:
+
+| PubRouter Metadata | DC terms | Description |
+|:-----------------------------:|:-------------------------:|:------------------------------------------------------------:|
+| journal.title | bibliographicCitation |  |
+| journal.abbrevTitle | bibliographicCitation |  |
+| journal.volume | bibliographicCitation |  |
+| journal.issue | bibliographicCitation |  |
+| journal.publisher | publisher |  |
+| journal.identifier.issn | source  | (issn: value) |
+| journal.identifier.eissn | source | (eissn: value) |
+| journal.identifier.pissn | source  | (pissn: value) |
+| journal.identifier.doi | source | (doi: value) |
+| article.title | title | A name given to the resource. |
+| article.subtitle | title | A name given to the resource. |
+| article.type | type | The nature or genre of the resource. |
+| article.version | description |  |
+| article.start_page | bibliographicCitation |  |
+| article.end_page | bibliographicCitation |  |
+| article.page_range | bibliographicCitation |  |
+| article.num_pages | ---- |   |
+| article.language | language |  |
+| article.abstract | abstract |  |
+| article.identifier.type | identifier  | (type: id) |
+| article.identifier.id | identifier  | (type: id) |
+| article.subject | subject |  |
+| author.type | creator |  |
+| author.name | creator |  |
+| author.organisation_name | creator |  |
+| author.identifier.orcid | creator |  |
+| author.identifier.email | creator |  |
+| author.affiliation | ---- |   |
+| contributor.type | contributor |  |
+| contributor.name | contributor |  |
+| contributor.organisation_name | contributor |  |
+| contributor.identifier.orcid | contributor |  |
+| contributor.identifier.email | contributor |  |
+| contributor.affiliation | ---- |   |
+| accepted_date | dateAccepted |  |
+| publication_date | issued | Date of formal issuance (e.g., publication) of the resource. |
+| history_date.type | description | History: {type} {date}, {type} {date}, ...' |
+| history_date.date | description | History: {type} {date}, {type} {date}, ...' |
+| history_date.date | dateSubmitted | Dspace uses dateSubmitted when history_date.type=="received"  |
+| publication_status | description |  |
+| project.name  | description | Funding information |
+| project.identifier | description | Funding information |
+| project.grant_number | description | Funding information |
+| embargo.start | rights |  |
+| embargo.end | rights |  |
+| embargo.duration | rights |  |
+| license_ref.title | rights |  |
+| license_ref.type | rights |  |
+| license_ref.url | rights |  |
+| license_ref.version | rights |  |
+| license_ref.start | rights |  |
```

Comment:
It isn't helpful to have separate rows for entries that are sent in a dcterms:rights element as a concatenated string.  In this case you should have just 1 row and explain the formatting in Description.

(Same comment applies elsewhere in similar circumstances).